containers/python-examples-redpanda/docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
     - PLAINTEXT://0.0.0.0:29092,OUTSIDE://0.0.0.0:9092
     - --advertise-kafka-addr
     - PLAINTEXT://redpanda:29092,OUTSIDE://localhost:9092
-    image: docker.vectorized.io/vectorized/redpanda:v21.4.13
+    image: docker.redpanda.com/vectorized:v21.4.13
```

Comment:
I don't think you want to remove the Redpanda name...

Reply:
Whoops, I'm a dummie